### ISSUE TITLE
Roslyn source generator: <IncludeSymbols> must be 'true'

### ISSUE THREAD

**Issue by JurJean** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#866]
When I configure this setting in the csproj of my source generator:

```
  <PropertyGroup>
    <IncludeSymbols>false</IncludeSymbols>
  </PropertyGroup>
```

I get the message: 

```
Proj0220: The <IncludeSymbols> must be 'true' when package format is 'snupkg' ([external_roslyn:Proj0220](https://sonarcloud.io/organizations/tjip/rules?open=external_roslyn%3AProj0220&rule_key=external_roslyn%3AProj0220) )
```

Is this a false positive, or is it valid and how should I fix it?

**Comment by Corniel** [comment]
This is a FP. The question is: how to detect a package that only contains analyzers or code generators. Is `<DevelopmentDependency>true</DevelopmentDependency>` sufficient.
